moduleroot/CONTRIBUTING.md.erb

```diff
@@ -206,6 +206,35 @@ SPEC_FACTS_OS=redhat bundle exec rspec spec/classes/foreman_spec.rb
 For more information on running the tests, see [rspec-puppet-facts](https://github.com/mcanevet/rspec-puppet-facts)
 and specifically the [section for running tests](https://github.com/mcanevet/rspec-puppet-facts#running-your-tests).
 
+**Running acceptance tests locally**
+
+1. Make sure you've met Prerequisites.
+
+2. Acceptance tests are running within a VM, so make sure you have installed:
+  - `docker` **or** 
+  - `vagrant` with `VirtualBox` as the provider **or**
+  - `vagrant` and `vagrant-libvirt` with `libvirt` as the provider
+
+For more dependencies please see related information for picked variant.
+Additonal info can be found at [voxpupuli-acceptance](https://github.com/voxpupuli/voxpupuli-acceptance/#running-tests) which is used to run the tests.
+
+3. For first (or clean) runs, run:
+  - `bundle exec rake spec_clean && bundle exec rake spec_prep`
+
+This will make sure all needed modules are prepared to be used in a VM.
+
+4. Run the tests:
+ - `BEAKER_HYPERVISOR=vagrant_libvirt bundle exec rspec`
```

Comment:
Weird, I've never run `rake beaker`, but `rspec` :/

Reply:
It might very well be my setup :)

Reply:
The intended command is `rake beaker` for the whole acceptance test suite. That already depends on `spec_prep` so that isn't a separate step.

If you just run `rspec` then you may also run unit tests.